wp-content/themes/workday/less/inc/header.less

```diff
@@ -0,0 +1,85 @@
+#site-header {
+    margin-top: 0;
+    margin-bottom: 0;
+    .header-container {
+        background: @black;
+        margin-left: calc(50% - 50vw);
+        width: 100vw;
+        height: 120px;
```

Comment:
@joshdarby re https://github.com/INN/umbrella-workdayminn/pull/19#issuecomment-516964647, it may make more sense to have the image height be set using the same CSS as this.

```css
#site-header {
    margin-top: 0;
    margin-bottom: 0;

    .header-container, img.attachment-home-logo {

        height: 120px;
    }
    .header-container {
        background: @black;
        margin-left: calc(50% - 50vw);
        width: 100vw;

Reply:
Wouldn't just `max-height: -webkit-fill-available;` make more sense here? Since we wouldn't want it to always be set to 120px.

Reply:
Besides it not working in non-webkit browsers, since the immediate parent is an `a` and not the container element, I don't think it'd work?

Reply:
I tried this:

```css
#site-header .header_img {
  height: available;
  height: -webkit-fill-available;
  box-sizing: border-box;
  padding: 0;
}
```

Reply:
That's fair. I used your suggestion from https://github.com/INN/umbrella-workdayminn/pull/19#discussion_r309367465 in https://github.com/INN/umbrella-workdayminn/pull/19/commits/71e9bde138791288889de37d15c6a6989d24b821.